readme.txt

```diff
@@ -34,6 +34,43 @@ make
 then as root
 make install
 
+
+Customize Desktop Environment for xRDP Session
+
+If you do not want to use the default desktop environment, you can customize it by creating a .Xclients file (X is capital!!!) in your home directory to launch the desktop environment you want and making it executable. In order to do this, open a terminal and run one of the following commands
+
+Gnome 3:
+echo "gnome-session" > ~/.Xclients
+chmod +x ~/.Xclients
```

Comment:
It is really needed on any distro? I prefer not to have executable configuration files in my home directory.